doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -2915,11 +2916,14 @@ Fields
 * "cyu": List of found CYUs in the packet
 * "cyu[].hash": CYU hash
 * "cyu[].string": CYU string
+* "ja3": The JA3 fingerprint consisting of both a JA3 hash and a JA3 string
+* "ja3s": The JA3S fingerprint consisting of both a JA3 hash and a JA3 string
+* "ja4": The JA4 client fingerprint for QUIC
```

Comment:
What about ja4s ?

Reply:
Mentioned in SV PR https://github.com/OISF/suricata-verify/pull/1634/files#diff-62a9c97571df60a1b3e213d1c5951c7e32eacba95631b69e35108561a97d1d5aR35

Reply:
JA4 (the client fingerprint is the only one currently available as open source:
> JA4: TLS Client Fingerprinting is open-source, BSD 3-Clause, same as JA3. FoxIO does not have patent claims and is not planning to pursue patent coverage for JA4 TLS Client Fingerprinting. This allows any company or tool currently utilizing JA3 to immediately upgrade to JA4 without delay.
> JA4S, JA4L, JA4H, JA4X, JA4SSH, and all future additions, (collectively referred to as JA4+) are licensed under the FoxIO License 1.1. This license is permissive for most use cases, including for academic and internal business purposes, but is not permissive for monetization. If, for example, a company would like to use JA4+ internally to help secure their own company, that is permitted. If, for example, a vendor would like to sell JA4+ fingerprinting as part of their product offering, they would need to request an OEM license from us.

That is basically the only reason I did not implement it right now.

Reply:
That's just literally the output from the reference implementation tool for documentation in the S-V README (which obviously implements JA4s). I can surely remove it but since we're not implementing it I don't see why mentioning a single specific hash there should be a problem.

Reply:
Just removed from S-V anyway :)